modules/zlib/1.3/patches/add_build_file.patch

```diff
@@ -0,0 +1,46 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,43 @@
++package(default_visibility = ["//visibility:public"])
++
++licenses(["notice"])  # BSD/MIT-like license (for zlib)
++
++cc_library(
++    name = "zlib",
++    srcs = [
++        "adler32.c",
++        "compress.c",
++        "crc32.c",
++        "crc32.h",
++        "deflate.c",
++        "deflate.h",
++        "gzclose.c",
++        "gzguts.h",
++        "gzlib.c",
++        "gzread.c",
++        "gzwrite.c",
++        "infback.c",
++        "inffast.c",
++        "inffast.h",
++        "inffixed.h",
++        "inflate.c",
++        "inflate.h",
++        "inftrees.c",
++        "inftrees.h",
++        "trees.c",
++        "trees.h",
++        "uncompr.c",
++        "zconf.h",
++        "zutil.c",
++        "zutil.h",
++    ],
++    hdrs = ["zlib.h"],
++    copts = select({
++        "@bazel_tools//src/conditions:windows": [],
```

Comment:
These conditions are being deprecated, could you use `@platforms//os:windows` instead?

Reply:
I copied now the zlib.BUILD file from https://github.com/protocolbuffers/protobuf/blob/master/third_party/zlib.BUILD which has this issue fixed.